config_utilities/test/tests/enums.cpp

```diff
@@ -196,7 +196,7 @@ TEST_F(Enums, enum_field) {
   // Getting fields.
   meta_data = internal::Visitor::getValues(config);
   EXPECT_EQ(meta_data.errors.size(), 0);
-  EXPECT_EQ(meta_data.data["reggular_enum"].as<std::string>(), "B");
+  EXPECT_EQ(meta_data.data["reggular_enum"].as<std::string>(), "BBB");
```

Comment:
All these changes may need a more thorough cleanup, depending on the whole "modifying global definition" discussion above. wdyt @Schmluk ?

Reply:
I'd suggest keeping the global definition behavior of enums unchanged for now. Happy to discuss other possible solutions more though!